<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>732</width>
    <height>456</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WEB CRAWLER</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PROJECT NAME:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="project_name">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>40</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>WEBSITE:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="home_page">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>100</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="start_crawling">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>691</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>START CRAWLING</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="output">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>691</width>
      <height>171</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>732</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
